submissions/akulahere/html-css-popup/index.html

```diff
@@ -0,0 +1,191 @@
+<!DOCTYPE html>
+<head>
```

Comment:
You are forget about tag `html`. When you will be adding it, don't forget about attribute `lang`.

Reply:
tag html is included in Doctype  @smillims